docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -633,6 +633,23 @@ Below is the list of the supported config properties.
   It should not be modified.
 </Admonition>
 
+### `tsh ssh` environment variables
+
+Under the hood, Teleport Connect uses `tsh ssh` to connect to Nodes. As a result,
```

Comment:
```suggestion
Under the hood, Teleport Connect uses `tsh ssh` to connect to SSH servers. As a result,
```

Within this doc page, "SSH node" is used only in the context of Connect My Computer, elsewhere we say "SSH server", e.g., https://goteleport.com/docs/connect-your-client/teleport-connect/#connecting-to-an-ssh-server